src/main/java/Core/Mains/MainClient2.java

```diff
@@ -0,0 +1,15 @@
+package Core.Mains;
+
+import Core.Client;
+
+import java.io.IOException;
+import java.net.Socket;
+
+public class MainClient2 {
```

Comment:
How come we need multiple `MainClient*.java`? You should be able to create a Jar from your project and run the jar multiple times against your server...

Paging @Gabbendorf to help with gradles and jars 😸